posts/2023-06-09-setting-up-a-home-energy-monitoring-system.adoc

```diff
@@ -0,0 +1,711 @@
+* How does the power used to charge the battery of an electrical bicycle evolve during a charging cycle?
+
+For the question, we can have a look at this graph which display the airco power consumption on April 7^th^ of this year:
+
+image::power-profile-airco.png[Power consumption profile for airco]
+
+After an initial spike (with a peak of 2kW) for about 20 minutes, the system evolves to a steady state where it consumes about 0.4kW for about 4 hours, after which changes to a kind of on-off mode.
+
+As for the second question, we can have a look at the following graph with data from May 15^th^:
+
+image::power-profile-electrical-bike-charging.png[Power consumption profile for charging electrical bicycle]
+
+We see that the power at which the battery is charged rises from an initial value of ±160W to 180W over a period of almost 2 hours. I think this is because it wasn't very warm outside, the battery's temperature rises during charging and the charging current is probably a function of that temperature, but that's an unconfirmed and personal theory. After reaching the peak, the charging slows down and stops after another one and a half hour.
+
+In conclusion, we can learn a lot about the different electrical consumers in the house and even learn a few special things about certain devices like the air conditioning.
+
+=== The Power Quality dashboard
+
+This dashboard contains two graphs that display the following data for all channels on the _Smappee Infinity_:
+
+* Apparent and Real power 
+* cos(&#966;)
+
+We can have a look at the data for the garage when the hybrid car is charged:
+
+image::charging-hybrid-car-power.png[Power consumption during charging of hybrid car]
+
+The apparent power and real power are 2365 VA and 2357 W respectively. This corresponds to a cos(&#966;) = 0.997 which is pretty much close to ideal. This is especially important as the power consumed during the charging is about 2.4kW. 
+
+On the other hand, the charger of our electrical bicycle charger's cos(&#966;) is only 0.69. So, when charging, the different powers are P~real~ = 180W, P~a~ = 260VA, and P~reactive~ = 187W. Even though it's "only 187W", when thousand or millions of devices with a low Power factor are online, the impact _is_ significant.
+
+<<<
+[id=cos-phi]
+=== Electrical Power Factor (also known as cos(&#966;))
+
+The Power Factor is a measure of an electrical system's energy utilisation efficiency. The power consumption can be decomposed in three parts:
+
+* P~a~: Apparent power, which is the product of the measured voltage and current.
+
+* P~real~: Effective- or Real Power is the part that actually produces work in the broad sense of the word, it's not the internal efficiency of the device itself. For example, an incandescent lightbulb consuming 100W converts 98W to heat and only 2W to light. So, in this case, the Productive Power is 100W (this is not to be confused with the efficiency of that bulb which is 2%).
```

Comment:
```suggestion
* P~real~: Effective or Real Power is the part that actually produces work in the broad sense of the word, it's not the internal efficiency of the device itself. For example, an incandescent lightbulb consuming 100W converts 98W to heat and only 2W to light. So, in this case, the Productive Power is 100W (this is not to be confused with the efficiency of that bulb which is 2%).
```
Do we need the `-` there?

Reply:
This is actually a literal transplantation of a language construct in Dutch:
In Dutch, instead of writing: _Effective Power or Real Power is_
we write _Effective**-** or Real Power is_ where the hyphen is a placeholder for _Power_ .
I just tried to find if English has something similar, but apparent it hasn't. I'll commit your suggestion.